.flake8

```diff
@@ -24,10 +24,10 @@
 #           produces false positives if you're surrounding things with double quotes
 
 [flake8]
+extend-select = B9
 max-line-length = 80
 max-complexity = 12
 noqa-require-code = true
-select = B,C,E,F,W,Y,B9,NQA
```

Comment:
If you have `select` specified in your flake8 config file, it clobbers `--extend-select`, even if `--extend-select` is passed on the command line.